snapshot.go

```diff
@@ -0,0 +1,38 @@
+package megos
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+// GetMetricsSnapshot will return the snapshot of pid
+func (c *Client) GetMetricsSnapshot(pid *Pid) (map[string]interface{}, error) {
```

Comment:
Would it make sense to add a new struct for the snapshot endpoint like you did for System?
With this we get rid of the interface{} and the necessary casting afterwards.
You applied the dummy json already in tests/master1.metrics.snapshot.json